Makefile

```
@@ -0,0 +1,37 @@
+ANSIBLE_VERBOSE?=
+
+ANSIBLE_EXTRA_FLAGS?=-v
+
+ve: ve/bin/activate
+ve/bin/activate: requirements.txt
+	@test -d ve || virtualenv ve
+	@ve/bin/pip install -Ur requirements.txt
+	@touch ve/bin/activate
+
+# Setup the VMs and the SSH config needed for Ansible to communicate to them.
+setup:
+	vagrant up tester-centos7-64
+	vagrant ssh-config tester-centos7-64 > ssh_config
+
+status:
+	vagrant status tester-centos7-64
+
+ansible-playbook-run-%: ve
+	OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES \
```

Do not you need the activation of the virtualenv?

```suggestion
	. ve/bin/activate: \
	OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES \
```